### ISSUE TITLE
Import should add, not overwrite

### ISSUE THREAD

**Issue by strider72** [bueltge/AddQuicktag#38]
I had a bunch of quicktags in Settings.  I tested an import file, and rather than adding the new tag, it completely erased anything that already existed, leaving **only** the imported tag.  It also saved the settings as such so there was no way to undo.

One of two things should change here:  Either give the user an explicit option to add or overwrite, OR the plugin should not immediately save the options, so the user can "undo" by leaving the Settings page without saving.

**Comment by bueltge** [comment]
Hi @strider72 
right, currently is the import a full overwrite of the settings, no checks for existing settings.

That I understand right. You will add new settings from a other installation, additional settings to current settings.

**Comment by strider72** [comment]
Correct.  When I import, it should **add** the new settings to existing settings. It should not remove the existing settings.
